recipes/blake3/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "blake3" %}
+{% set version = "0.1.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99d688bd1b673de332a2927665e0e82eefae79b18b7c03564c5907d716e473b8
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - maturin
+    - pip
+    - python
+    - rust
```

Comment:
Can this be noarch:python? Or no because of the rust?

Reply:
Right, this library is bindings for the underlying Rust code, so the build artifacts are arch-specific.

Reply:
Got it, cool